### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by 972821054** [fzh0917/STMTrack#28]
Hello, I would like to ask about the loss of return, what is the difference between iou and reg here? And during the training process, when did your reg and iou loss converge? It is still convergent, and I see that iou damage does not backpropagate, what is its function?
![image](https://user-images.githubusercontent.com/83146207/225541307-5a6e1f53-85ca-4dc5-a4e1-1ea684cc495c.png)
